src/bp-core/classes/class-bb-presence.php

```diff
@@ -268,7 +268,7 @@ public static function bb_get_users_last_activity( $user_ids ) {
 		 *
 		 * @since BuddyBoss 2.3.1
 		 */
-		public static function bb_load_presence_api_mu_plugin( $bypass = true ) {
+		public static function bb_load_presence_api_mu_plugin( $bypass = false ) {
```

Comment:
@rajinsharwar2 
Pass bypass as false where we are using this function, instead of directly here.